development-updates.md

```diff
@@ -110,7 +110,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
 | [Richa](https://github.com/Richa-iitr)                       | [Update 3](https://hackmd.io/@iri/rJrPvd08C)                    | [Update 4](https://hackmd.io/@iri/rkbRsVtPA)                    | [Update 5](https://hackmd.io/@iri/HkJ2kR4O0)                    |                                                                                                                                                        |
 | [Rodrigo Herrera](https://github.com/)                       | [Update 3](https://hackmd.io/@rodrigoh/update3)                 |                                                                 |                                                                 |                                                                                                                                                        |
 | [RoryArredondo](https://github.com/arredr2)                  | [Update 3](https://hackmd.io/@arredr2/Bkehft2IR)                | [Update 4](https://hackmd.io/@arredr2/BJ_GbXzDA)                |                                                                 |                                                                                                                                                        |
-| [Rose Jethani](https://github.com/rose2221)                  | [Update 3](https://hackmd.io/@0xrosetteeee/Week3)               | [Update 4](https://hackmd.io/@0xrosetteeee/Week4)               | [Update 5](https://hackmd.io/@0xrosetteeee/BJw067Eu0)           |                                                                                                                                                        |
+| [Rose Jethani](https://github.com/rose2221)                  | [Update 3](https://hackmd.io/@0xrosetteeee/Week3)               | [Update 4](https://hackmd.io/@0xrosetteeee/Week4)               | [Update 5](https://hackmd.io/@0xrosetteeee/BJw067Eu0)           | [Prysm libP2P Implementation](https://hackmd.io/@iri/r1q1oV5PC)                                                                                                                                                       |
```

Comment:
The project needs to be in `/projects` directory, submitted by PR and reviewed

Reply:
oh my bad, I'm closing this PR and will open a new one